### PR TITLE
sig-observability charter update

### PR DESCRIPTION
**Pull request by halcyondude**

### CODE REVIEW

File: sigs/observability.md [hunk 1]
```diff
@@ -156,13 +160,13 @@ Examples include:
 - Security aspects that need to be present when setting up cloud native
   infrastructure, these might be more relevant for SIG Security.
 - How cloud native applications that need observability are deployed; this would
-  fall in the scope of SIG App Delivery
+  fall in the scope of SIG App Delivery.`
```
Comment (amye) [inline sigs/observability.md:163]: This looks like an error?

File: sigs/observability.md [hunk 2]
```diff
@@ -156,13 +160,13 @@ Examples include:
...
 - Tools and projects that are used to run cloud native workloads that in some
   cases need observability would fall under the scope of SIG-Runtime.
 
 ## Roadmap & Initial Efforts
 
-- Contribute to [due diligence reports][ddr] to assist the CNCF TOC for projects
+- Contribute to [due diligence reports][ddg] to assist the CNCF TOC for projects
```
Comment (amye) [inline sigs/observability.md:169]: Guidelines or reports?

File: sigs/observability.md [hunk 3]
```diff
@@ -183,20 +187,24 @@ Examples include:
...
 [sigroles]: https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#sig-member-roles
+[sigtlprocess]: https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#elections
 
 - Contact
   - Slack channel: #sig-observability @ [https://cloud-native.slack.com](https://cloud-native.slack.com)
   - Email List: [cncf-sig-observability@lists.cncf.io](mailto:cncf-sig-observability@lists.cncf.io)
 - Meeting Schedule:
   - TBD - pending feedback from SIG members
-  - [https://www.cncf.io/community/calendar](https://www.cncf.io/community/calendar/)
+  - [https://www.cncf.io/community/calendar](https://www.cncf.io
```
Comment (amye) [inline sigs/observability.md:210]: This looks like a link that won't resolve.

**Review by amye — CHANGES_REQUESTED** [review]

File: sigs/observability.md (hunk 2 above)
Comment (halcyondude) [reply sigs/observability.md:169]: Due Diligence Reviews/Reports.  update incoming.

File: sigs/observability.md (hunk 3 above)
Comment (halcyondude) [reply sigs/observability.md]: fixed

**Comment by halcyondude** [comment]
Thanks for feedback!  Apologies for response delay.

File: sigs/observability.md (hunk 3 above)
Comment (amye) [reply sigs/observability.md]: Still off?

**Comment by halcyondude** [comment]
@amye thanks for feedback, resolved broken links.

**Comment by halcyondude** [comment]
Closing this, merged changes to a new clean PR 

https://github.com/cncf/toc/pull/660

**Pull request closed without merging**